addons/mail/views/discuss/discuss_gif_favorite_views.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<odoo>
+
+    <record id="discuss_gif_favorite_view_form" model="ir.ui.view">
+        <field name="name">discuss.gif.favorite.form</field>
+        <field name="model">discuss.gif.favorite</field>
+        <field name="arch" type="xml">
+            <form string="Gifs favorite">
+                <sheet>
+                    <group>
+                        <field name="tenor_gif_id"/>
```

Comment:
We have enough space, it would nice to have id and user id displayed too.

Reply:
Id and creator are displayed by default:
![image](https://github.com/odoo/odoo/assets/1810149/78a5d95e-75ef-493b-8c15-c49123b42c60)

Reply:
No fields are displayed by default (unless the view not is specified at all then the default is used).

The comment was on the form view, where those fields are currently not displayed.

The list view is indeed correct.